smi/internal/module.go

```diff
@@ -312,6 +321,19 @@ func GetModule(name string) (*Module, error) {
 	return LoadModule(name)
 }
 
+func LoadModuleFromStream(data []byte) (*Module, error) {
+	r := bytes.NewReader(data)
+	in, err := parser.Parse(r)
+	if err != nil {
+		return nil, errors.Wrap(err, "Parse module")
+	}
+	out, err := BuildModule("", in)
```

Comment:
does the module need a "name"?

Reply:
SmiModule has a `Name` field, and gosmi internally keeps a map from the name to module for quick lookup